testsuite/features/secondary/proxy_traditional_cobbler_pxeboot.feature

```diff
@@ -127,6 +127,14 @@ Feature: PXE boot a terminal with Cobbler and traditional proxy
     Then I should see a "1 package install has been scheduled" text
     When I wait until event "Package Install/Upgrade scheduled by admin" is completed
 
+  Scenario: Download the profile from the UI
+    When I follow the left menu "Systems > Autoinstallation > Profiles"
+    And I follow "15-sp4-cobbler"
+    And I follow "Autinstallation File" in the content area
+    And I follow "Download Autoinstallation file"
+    Then I should see a "<profile " text
+    And I go back
```

Comment:
Not needed, according to my tests (next scenario works fine)

Reply:
@srbarrios do you have more insights on that one or are we good to just delete that line?